packages/core-mobile/app/new/features/swap/hooks/useEvmSwap.ts

```diff
@@ -64,18 +61,9 @@ export const useEvmSwap = (): {
       abortControllerRef.current = controller
 
```

Comment:
I would also move this abort controller logic inside ParaswapProvider since it only applies there